hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -507,115 +508,129 @@ spec = around (showLogsOnFailure "EndToEndSpec") $ do
       it "does report on unsupported era" $ \tracer -> do
         pendingWith "Currently supporting Conway era no future upcoming"
         withClusterTempDir $ \tmpDir -> do
-          args <- setupCardanoDevnet tmpDir
-          forkIntoConwayInEpoch tmpDir args 1
-          withCardanoNode (contramap FromCardanoNode tracer) tmpDir args $ \node@RunningNode{nodeSocket} -> do
-            let hydraTracer = contramap FromHydraNode tracer
-            hydraScriptsTxId <- publishHydraScriptsAs node Faucet
-            chainConfig <- chainConfigFor Alice tmpDir nodeSocket hydraScriptsTxId [] cperiod
-            withHydraNode' hydraTracer chainConfig tmpDir 1 aliceSk [] [1] Nothing $ \out stdErr ph -> do
-              -- Assert nominal startup
-              waitForLog 5 out "missing NodeOptions" (Text.isInfixOf "NodeOptions")
-
-              waitUntilEpoch tmpDir args node 1
-
-              waitForProcess ph `shouldReturn` ExitFailure 1
-              errorOutputs <- hGetContents stdErr
-              errorOutputs `shouldContain` "Received blocks in unsupported era"
-              errorOutputs `shouldContain` "upgrade your hydra-node"
+          maybeRunningNode <- findRunningCardanoNode tmpDir
+          case maybeRunningNode of
+            Just _ -> fail "can't run forking e2e specs on running cardano-node"
+            Nothing -> do
```

Comment:
Must: not need this check

There is no expectation that these tests would run against an already running cardano node and the `tmpDir` is also randomly selected. Hence, these should just work as before!

Also applies to all changes in this file.